src/eoapi/vector/eoapi/vector/sql/functions.sql

```diff
@@ -132,3 +133,55 @@ BEGIN
     RETURN;
 END;
 $$ LANGUAGE PLPGSQL;
+
+CREATE OR REPLACE FUNCTION pg_temp.pgstac_hash_count(
+    IN queryhash text,
+    IN bounds geometry DEFAULT ST_MakeEnvelope(-180,-90,180,90,4326),
+    IN depth int DEFAULT 1,
+    OUT geom geometry,
+    OUT cnt bigint
+) RETURNS SETOF RECORD AS $$
+DECLARE
+    search record;
+    xmin float := ST_XMin(bounds);
+    xmax float := ST_XMax(bounds);
+    ymin float := ST_YMin(bounds);
+    ymax float := ST_YMax(bounds);
+    w float := (xmax - xmin) / depth;
+    h float := (ymax - ymin) / depth;
+    q text;
+BEGIN
+    SELECT * INTO search FROM pgstac.searches WHERE hash=queryhash;
+    DROP VIEW IF EXISTS searchitems;
+    EXECUTE format($q$
+        CREATE TEMP VIEW searchitems AS
+        SELECT geometry
+        FROM pgstac.items WHERE %s
+            AND ST_Intersects(geometry, %L)
+        ;
+        $q$,
+        search._where,
+        bounds
+    );
+    RETURN QUERY
+        WITH grid AS (
+            SELECT
+                ST_MakeEnvelope(
+                    xmin + w * (a-1),
+                    ymin + h * (b-1),
+                    xmin + w * a,
+                    ymin + h * b,
+                    4326
```

Comment:
is the bounds from the vector tile request always in EPSG:4326?

Reply:
👍 https://github.com/developmentseed/tipg/blob/82691b808c1265ca0b21b7cacf122880b426e2b5/tipg/dbmodel.py#L544-L546